sklearn/svm/_classes.py

```diff
@@ -669,6 +668,10 @@ class SVC(BaseSVC):
         Degree of the polynomial kernel function ('poly').
         Must be non-negative. Ignored by all other kernels.
 
+        .. deprecated:: 1.6
```

Comment:
I'm not sure a `deprecated` directive here is apt. We're not deprecating the parameter. Probably a `versionchanged` makes more sense?

Reply:
I agree with ```versionchanged```